codebuild/bin/s2n_codebuild.sh

```diff
@@ -13,12 +13,9 @@
 # permissions and limitations under the License.
 #
 
-set -ex
+set -e
 
 source codebuild/bin/s2n_setup_env.sh
-# Defer overriding LD_LIBRARY_PATH so that the overriden paths don't interfere with test install scripts.
-# Some Test install scripts use curl commands to download files from S3, but those commands don't work when forced to load OpenSSL 1.1.1
-source codebuild/bin/s2n_override_paths.sh
```

Comment:
Why is this no longer necessary?

Reply:
It messes up LD_LIBRARY_PATH.  The PATH over-rides were moved into s2n_setup_env.sh